ElectronClient/gui/note-viewer/index.html

```diff
@@ -132,6 +132,13 @@
 					console.warn('Cannot find hash', hash);
 					return;
 				}
+
+				// HACK: force :target pseudo class to take effect,
+				// so that user can use :target selector in userstyle.css
+				history.pushState({}, "", `#${hash}`);
+				history.pushState({}, "", `#${hash}`);
+				history.back();
```

Comment:
Getting scroll to work and jumping to anchors is often hard to get right and breaks easily, so I'm concerned this will cause other subtle issues. Is there any other way to make it work without hack?

Reply:
I'm sorry for replying late. I can't figure out other way to implement this feature and can't promise that this hack won't make any other bugs. Just feel free to close this PR if hacks are unacceptable.

Reply:
Thanks for your reply, and indeed I'm a bit worried that the fix might break something else. We lack test units in this part of the code to be confident about those things. So indeed I would prefer to close, and thanks for your understanding.